src/common/common.css

```
@@ -171,6 +173,21 @@ a:active {
   margin-top: 8px;
 }
 
+@media screen and (max-width: 767px) {
+  .message-box {
+    flex-wrap: wrap;
+  }
+  .message-text {
+    display: inline-block;
+  }
+
+  .message-link {
```

```suggestion
  .message > a {
```

shoudl work, too should not it?
(then we don't need to add another CSS class, as you did in the HTMl change below)

---

it should work but if in future a new a tag is added in anywhere this class is used it will have to override, I can do the same if needed?

---

the HTML structure is expected to be the same, also if it is just a sibling element it will continue to work; just if it is nested, it will fail

As such, please adjust it.

---

changes done